definitions/output/all/requests_stable.js

```diff
@@ -0,0 +1,176 @@
+        },
+      },
+      response_body: "Text-based response body",
+    },
+    tags: ["requests_stable"],
+}).preOps(ctx => `
+CREATE TABLE IF NOT EXISTS ${ctx.self()}
+(
+  date DATE NOT NULL OPTIONS(description="YYYY-MM-DD format of the HTTP Archive monthly crawl"),
+  client STRING NOT NULL OPTIONS(description="Test environment: desktop or mobile"),
+  page STRING NOT NULL OPTIONS(description="The URL of the page being tested"),
+  is_root_page BOOL OPTIONS(description="Whether the page is the root of the origin."),
+  root_page STRING NOT NULL OPTIONS(description="The URL of the root page being tested"),
+  rank INT64 OPTIONS(description="Site popularity rank, from CrUX"),
+  url STRING NOT NULL OPTIONS(description="The URL of the request"),
+  is_main_document BOOL NOT NULL OPTIONS(description="Whether this request corresponds with the main HTML document of the page, which is the first HTML request after redirects"),
+  type STRING OPTIONS(description="Simplified description of the type of resource (script, html, css, text, other, etc)"),
+  index INT64 OPTIONS(description="The sequential 0-based index of the request"),
+  payload STRING OPTIONS(description="JSON-encoded WebPageTest result data for this request"),
+  summary STRING OPTIONS(description="JSON-encoded summarization of request data"),
+  request_headers ARRAY<STRUCT<name STRING OPTIONS(description="Request header name"), value STRING OPTIONS(description="Request header value")>> OPTIONS(description="Request headers"),
+  response_headers ARRAY<STRUCT<name STRING OPTIONS(description="Response header name"), value STRING OPTIONS(description="Response header value")>> OPTIONS(description="Response headers"),
+  response_body STRING OPTIONS(description="Text-based response body")
+)
+PARTITION BY date
+CLUSTER BY client, is_root_page, type, rank
+OPTIONS(
+  require_partition_filter=true
+);
+
+CREATE TEMP FUNCTION PRUNE_OBJECT(
+  json_str STRING,
+  keys_to_remove ARRAY<STRING>
+) RETURNS STRING
+LANGUAGE js AS """
+  try {
+    var jsonObject = JSON.parse(json_str);
+    keys_to_remove.forEach(function(key) {
+      delete jsonObject[key];
+    });
+    return JSON.stringify(jsonObject);
+  } catch (e) {
+    return json_str;
+  }
+""";
+    `).query(ctx => `
+SELECT
+  requests.date,
+  requests.client,
+  requests.page,
+  requests.is_root_page,
+  requests.root_page,
+  crux.rank,
+  requests.url,
+  requests.is_main_document,
+  requests.type,
+  requests.index,
+  requests.payload,
+  PRUNE_OBJECT(
+      requests.summary,
+      ["crawlid", "firstHtml", "firstReq", "pageid", "req_accept_encoding", "req_accept_language", "req_accept", "req_if_modified_since", "req_if_none_match", "req_referer", "req_user_agent", "reqOtherHeaders", "requestid", "resp_age", "resp_cache_control", "resp_content_length", "resp_content_type", "resp_date", "resp_etag", "resp_last_modified", "resp_server", "resp_vary", "respOtherHeaders", "startedDateTime", "url", "urlShort"]) as summary,
```

Comment:
Here is a cleaned object example:
```json
{
    "time": 0,
    "_cdn_provider": null,
    "_gzip_save": null,
    "method": "GET",
    "reqHeadersSize": 598,
    "reqBodySize": null,
    "reqCookieLen": 0,
    "status": -1,
    "respHttpVersion": "HTTP/2",
    "redirectUrl": null,
    "respHeadersSize": null,
    "respBodySize": null,
    "respSize": 0,
    "mimeType": "",
    "ext": "js",
    "type": "script",
    "format": "",
    "respCookieLen": 0,
    "expAge": 0
}
```
I think we can also remove `type`, as it's available as a separate column.

```suggestion
      ["crawlid", "firstHtml", "firstReq", "pageid", "req_accept_encoding", "req_accept_language", "req_accept", "req_if_modified_since", "req_if_none_match", "req_referer", "req_user_agent", "reqOtherHeaders", "requestid", "resp_age", "resp_cache_control", "resp_content_length", "resp_content_type", "resp_date", "resp_etag", "resp_last_modified", "resp_server", "resp_vary", "respOtherHeaders", "startedDateTime", "type", "url", "urlShort"]) as summary,
```